src/models/user.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const sequelize = require('../dataBase');
+
+const User = sequelize.define('User', {
+  id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  createdAt: {
+    type: DataTypes.DATE,
+    field: 'created_at',
+    allowNull: false,
+    defaultValue: DataTypes.NOW,
+  },
+}, {
+  tableName: 'users',
+  updatedAt: false,
+});
+
+function normalize({ id, name }) {
+  return {
+    id, name,
+  };
+};
```

Comment:
it's rather a service, so move it to the service file

Reply:
done